actionview/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Rails 6.0.0.beta1 (January 18, 2019) ##
 
+*   Ensure unique DOM IDs for collection inputs with float values.
```

Comment:
I think this should go above the Rails 6 beta release line, because that already happened. Eg. https://github.com/rails/rails/pull/35033/files#diff-1846d4882fa63eb5b1271541995cbfdcR1

Reply:
Thanks - moved and force pushed.